README.md

```diff
@@ -140,12 +147,18 @@ sudo -E image
 Using `osbuilder` with ubuntu 14.06 fails because an old version of rpm. You can still use it
 exporting `USE_DOCKER` variable see [how to use docker](#Using-osbuilder-scripts-with-docker).
 
+
+## Environment Variables
+
+* `IMG_SIZE`: Image size. By default this value is 50M.
+* `EXTRA_PKGS`: The list of extra packages to install separated by spaces, for example "a b c". By default this values is empty.
```

Comment:
Have the "By default..." sentence start on a new line with indentation and clarify values vs value:
this value is empty
these values are empty